src/main/java/net/commments/object/BCommentPage.java

```diff
@@ -21,7 +21,19 @@ public void clickNewComment() {
         this.driver.manage().timeouts().implicitlyWait(15, TimeUnit.SECONDS);
     }
 
-    public int commentPagination() {
-        throw new UnsupportedOperationException("Please implement");
+    public void navigateToLastPage() {
+        String s = this.driver.findElement(By.xpath("//td[@colspan='5']")).getText();
+        System.out.println(s);
+        String[] pages = s.split(" ");
+        for (String s1 : pages)
+            System.out.println(s1);
+        String s2 = pages[pages.length - 2];
+        System.out.println(s2);
+        String page = s2;
+        this.driver.findElement(By.xpath(String.format("//a[@href='/?page=%s']", page))).click();
```

Comment:
Use `pages[pages.length - 2]` instead of `page`. And remove 3 previous lines.

Reply:
done